src/model/immutable/__tests__/CharacterMetadata-test.js

```diff
@@ -14,104 +14,99 @@
 
-var CharacterMetadata = require('CharacterMetadata');
-var {BOLD, BOLD_ITALIC, NONE, UNDERLINE} = require('SampleDraftInlineStyle');
-
-describe('CharacterMetadata', () => {
-  it('must have appropriate default values', () => {
-    var character = CharacterMetadata.create();
-    expect(character.getStyle().size).toBe(0);
-    expect(character.getEntity()).toBe(null);
-  });
+const CharacterMetadata = require('CharacterMetadata');
+const {BOLD, BOLD_ITALIC, NONE, UNDERLINE} = require('SampleDraftInlineStyle');
 
-  describe('Style handling', () => {
-    var plain = CharacterMetadata.create();
-    var bold = CharacterMetadata.create({style: BOLD});
-    var fancy = CharacterMetadata.create({style: BOLD_ITALIC});
-
-    it('must run `hasStyle` correctly', () => {
-      expect(plain.hasStyle('BOLD')).toBe(false);
-      expect(bold.hasStyle('BOLD')).toBe(true);
-      expect(fancy.hasStyle('BOLD')).toBe(true);
-      expect(plain.hasStyle('ITALIC')).toBe(false);
-      expect(bold.hasStyle('ITALIC')).toBe(false);
-      expect(fancy.hasStyle('ITALIC')).toBe(true);
-    });
-
-    it('must apply style', () => {
-      var newlyBold = CharacterMetadata.applyStyle(plain, 'BOLD');
-      expect(newlyBold.hasStyle('BOLD')).toBe(true);
-      var alsoItalic = CharacterMetadata.applyStyle(newlyBold, 'ITALIC');
-      expect(alsoItalic.hasStyle('BOLD')).toBe(true);
-      expect(alsoItalic.hasStyle('ITALIC')).toBe(true);
-    });
-
-    it('must remove style', () => {
-      var justBold = CharacterMetadata.removeStyle(fancy, 'ITALIC');
-      expect(justBold.hasStyle('BOLD')).toBe(true);
-      expect(justBold.hasStyle('ITALIC')).toBe(false);
-      var justPlain = CharacterMetadata.removeStyle(justBold, 'BOLD');
-      expect(justPlain.hasStyle('BOLD')).toBe(false);
-      expect(justPlain.hasStyle('ITALIC')).toBe(false);
-    });
-  });
+const plain = CharacterMetadata.create();
+const bold = CharacterMetadata.create({style: BOLD});
+const fancy = CharacterMetadata.create({style: BOLD_ITALIC});
 
-  describe('Entity handling', () => {
-    var withoutEntity = CharacterMetadata.create();
-    var withEntity = CharacterMetadata.create({entity: 'a'});
-
-    it('must apply entity correctly', () => {
-      var newKey = 'x';
-      var modifiedA = CharacterMetadata.applyEntity(withoutEntity, newKey);
-      var modifiedB = CharacterMetadata.applyEntity(withEntity, newKey);
-      expect(modifiedA.getEntity()).toBe(newKey);
-      expect(modifiedB.getEntity()).toBe(newKey);
-    });
-
-    it('must remove entity correctly', () => {
-      var modifiedA = CharacterMetadata.applyEntity(withoutEntity, null);
-      var modifiedB = CharacterMetadata.applyEntity(withEntity, null);
-      expect(modifiedA.getEntity()).toBe(null);
-      expect(modifiedB.getEntity()).toBe(null);
-    });
-  });
+const withoutEntity = CharacterMetadata.create();
+const withEntity = CharacterMetadata.create({entity: 'a'});
+
+const empty = CharacterMetadata.create();
```

Comment:
This is the same as 'plain' above.

I get that you were just copying out from the existing tests though.